src/context/WorkspaceContext/placeholder.ts

```diff
@@ -32,7 +32,7 @@ export const accountsPlaceholder = {
                     store_id: '3f7a9c1b-0d4e-4f2a-b3d2-c8e9f2d4a7e1',
                     name: 'sample-test-AlexSmith-storage',
                     access_point_id: 'fsap-0123abc456d789012',
-                    fs_id: 'fs-1234abcd5678efgh',
+                    mount_point: 'fs-1234abcd5678efgh',
```

Comment:
I can see the response from `/api/workspaces` is now `"mount_point": "/workspaces/workflowtestuser"`